doc/nrf/security/tfm/tfm_logging.rst

```diff
@@ -14,17 +14,21 @@ By default, the logs arrive on different COM ports on the host PC.
 Configuring logging
 *******************
 
-You can configure the TF-M to connect to the same UART as the application with the :kconfig:option:`CONFIG_TFM_SECURE_UART0` Kconfig option.
-Setting this Kconfig option makes TF-M logs visible on the application's VCOM, without manual connection.
+You can configure TF-M to connect to the same UART as the application.
+Setting the appropriate Kconfig option makes TF-M logs visible on the application's VCOM, without manual connection.
 
-The UART instance used by the application is ``0`` by default, and the TF-M UART instance is ``1``.
-To change the TF-M UART instance to the same as that of the application's, use the :kconfig:option:`CONFIG_TFM_SECURE_UART0` Kconfig option.
+The UART instances can vary by device family:
 
-When building TF-M with logging enabled, UART1 must be disabled in the non-secure application, otherwise the non-secure application will fail to run.
+* nRF5340 and nRF91 Series: The application typically uses UART instance ``0``, and TF-M uses UART instance ``1`` by default.
+* nRF54L Series: The application typically uses UART instance ``20`` (``uart20``), and TF-M uses a different UART instance by default (check the devicetree specification for the board target you are using).
+
+To configure TF-M to connect to the same UART as the application, you can use the ``CONFIG_TFM_SECURE_UART`` Kconfig options, for example :kconfig:option:`CONFIG_TFM_SECURE_UART0` or :kconfig:option:`CONFIG_TFM_SECURE_UART20` for the examples above.
```

Comment:
I tried to set CONFIG_TFM_SECURE_UART20=y for nRF54L15 but get compile errors since UART30 is hard-coded as secure uart for TF-M in cmake:
modules/tee/tf-m/trusted-firmware-m/platform/ext/target/nordic_nrf/common/nrf54l/config.cmake

Reply:
Is there a way to have TF-M logs visible on nRF54L15 then?